README.md

```diff
@@ -1,12 +1,24 @@
+![MageLogo](https://user-images.githubusercontent.com/99209078/171298628-d8f34d5b-9771-404d-947c-f446ce083215.png)
 # Mage AI
+Mage AI cleans your data using a single Python library.
 
 ## Table of Contents
 1. [Introduction](#introduction)
 1. [Installation](#installation)
 1. [Getting Started](#getting-started)
+2. [Usage](#usage)
+3. [To Contribute](#to-contribute)
 
 ## Introduction
-Mage AI is a Python library for data cleaning.
+Mage AI aims to simplify data pre-processing. With a few lines of code, you can:
+- Generate data cleaning suggestions in your local notebook
+- Display feature-specific statistics and visualizations
+- Easily detect "holes" and anomalies in your data
+
+Coming soon:
+- Access faster data analysis and cleaning using Mage's public cloud server
+- Perform no-code data transformations within the UI
+- Export cleaned data into your ML pipeline
```

Comment:
Support user defined cleaning functions within the UI.

Reply:
Update "Export cleaned data into your ML pipeline" to "Export data cleaning pipeline and integrate into your production data pipelines."